### DISCUSSION TITLE
Change Theme

### DISCUSSION THREAD

**Discussion by JEEVAN28** [YetiForceCompany/YetiForceCRM#13852] [How to]
I am referring https://github.com/YetiForceCompany/YetiForceCRM/issues/2950 but not able to find exact file, i tried to change style.css from twilite folder but does not work.

please assist in the matter.

Thanks in advance

Regards
Jeevan Pendhari

**Comment by davide-alghi** [comment]
Hi @JEEVAN28,
because ```public static $MINIMIZE_CSS = true;```
in file **config/Developer.php**, YF loads **style.min.css** file, so your changes are not taken into account.
Setting ```public static $MINIMIZE_CSS = false;``` your changes will be loaded but the system performance will slightly slow down
